src/Components/SmartComponents/SystemsExposedTable/SystemsExposedTableToolbar.js

```diff
@@ -135,7 +135,8 @@ export const SystemsExposedTableToolbar = ({
             }}
             expandAll={hasSecurityRule ? {
                 isAllExpanded,
-                onClick: () => expandAll(rawData, isAllExpanded)
+                onClick: () => expandAll(rawData, isAllExpanded),
+                isDisabled: meta.totalItems === 0
```

Comment:
Just a minor thing: Should we handle the case with `meta?.totalItems ===0` when meta might be undefined?

Reply:
Good point, `totalItems` is in the `meta` schema of the API response, so we can trust the backend that param will always be present.